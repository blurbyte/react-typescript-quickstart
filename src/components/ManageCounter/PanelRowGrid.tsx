import styled from '@styles/styledComponents';

const PanelRowGrid = styled.div`
  display: grid;
  grid-template-columns: repeat(2, 1fr);
  grid-gap: 1rem;
`;

export default PanelRowGrid;
