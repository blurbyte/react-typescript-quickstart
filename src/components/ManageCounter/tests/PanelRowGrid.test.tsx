import * as React from 'react';
import { shallow } from 'enzyme';

import PanelRowGrid from '../PanelRowGrid';

describe('PanelRowGrid', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<PanelRowGrid />);
    expect(wrapper).toMatchSnapshot();
  });
});
