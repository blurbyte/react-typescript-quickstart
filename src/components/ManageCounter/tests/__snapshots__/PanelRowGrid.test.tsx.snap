// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PanelRowGrid renders correctly 1`] = `
.c0 {
  display: grid;
  grid-template-columns: repeat(2,1fr);
  grid-gap: 1rem;
}

<div
  className="c0"
/>
`;
